### PR TITLE
RubyK_Assignment_27

### PR DESCRIPTION
**Pull request by rubykoct2020**
image

### CODE REVIEW

File: src/ruby/NumberInString.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+	void findNumberInString(String[] input) {
+		System.out.println("Program 1: Output");
+		for (int index = 0; index < input.length; index++) {
+			String str = input[index];
+			boolean flag = false;
+			for (int i = 0; i < str.length(); i++) {
+				char ch = str.charAt(i);
+				if (Character.isDigit(ch)) {
+					flag = true;
+				}
```
Comment (harshvegada) [inline src/ruby/NumberInString.java:21]: you should break once you know that string contains digit

File: src/ruby/NumberInString.java [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+			String str = input[index];
+			String temp = "";
+			int sum = 0;
+			for (int i = 0; i < str.length(); i++) {
+				char ch = str.charAt(i);
+				if (Character.isDigit(ch)) {
+					temp += ch;
+				}
+				if (!temp.equals("")) {
+					sum += Integer.parseInt(temp);
```
Comment (harshvegada) [inline src/ruby/NumberInString.java:46]: you can use Character.getNumericValue(ch) method right,
you need to add single char value and do sum

Comment (harshvegada) [reply src/ruby/NumberInString.java:46]: I hope will clear on above method, take of this next
No logical change require

**Review by harshvegada — APPROVED** [review]
Code matches require output!
Code logically correct!
Good to merge!

File: src/ruby/NumberInString.java (hunk 2 above)
Comment (rubykoct2020) [reply src/ruby/NumberInString.java:46]: Thanks Harsh for you comments. I will correct and push it again

**Pull request merged**